lib/contentful.js

```diff
@@ -6,15 +6,17 @@
 const contentful = require('contentful');
 const Promise = require('bluebird');
 const logger = require('winston');
-
+const config = require('../config/contentful');
 const helpers = require('./helpers');
 const UnprocessibleEntityError = require('../app/exceptions/UnprocessibleEntityError');
 
-const defaultCampaignId = process.env.CONTENTFUL_DEFAULT_CAMPAIGN_ID || 'default';
+const defaultCampaignId = config.defaultCampaignId;
 
 // The sequence we define peroperties here determines the order they appear in GET Gambit Campaigns
 // API response. Should match the sequence defined in Contentful, which is based on the order in
 // which our end user will see the messages.
+// TODO: Javascript Objects do not guarantee order,
+// we should change this to an array if order is important.
```

Comment:
The order is only really used by the Slothbot "View Messages" functionality -- so order isn't critical. I think it's safe to leave as an object for now, as it's an easy way to map the machine name style variables with the corresponding Contentful field name.